[WIP] measurement level `1` for T1 experiment (#857)
Qiskit/qiskit-experiments: docs/tutorials/t1.rst

        for result in sub_data.analysis_results():
            print(result)

:math:`T_1` experiments with kerneled measurement
---------------------------------------------------

> :math:`T_1` experiment can also be done with kerneled measurement.

Reviewer: ```suggestion
:math:`T_1` experiments can also be done with kerneled measurements.
```

Author: Changed as suggested.

> If we will set the running option `meas_level=MeasLevel.KERNELED`, the

Reviewer: ```suggestion
If we set the runn option `meas_level=MeasLevel.KERNELED`, the
```

Author: Changed as suggested (without the typo).
```If we set the run option `meas_level=MeasLevel.KERNELED`, the job```

> job will not discriminate the data and will not label it. In T1 experiment,

Reviewer: ```suggestion
job will not discriminate the data and will not label it. In the T1 experiment,
```

Author: Changed as suggested.

> as we know that in :math:`P(1|t=0)=1`, we will add measurement for delay=0

Reviewer: ```suggestion
since we know that :math:`P(1|t=0)=1`, we will add a circuit with delay=0,
```

Author: Changed as suggested.

> since we know that in :math:`P(1|t=0)=1`, we will add a circuit with delay=0,

Author: Changed as suggested.

> and for extreme delay. In this configuration we know that the data starts from

Reviewer: ```suggestion
and another circuit with a very large delay. In this configuration we know that the data starts from
```

Author: Changed as suggested.

a point [I,Q] that is close to a logical value '1' and ends at a point [I,Q]

> that is close to a logical value '0'. Beside that, We will need to use analysis
> option to normalize the output.

Reviewer: ```suggestion
that is close to a logical value '0'. 
```

The analysis option is added automatically.

Author: Changed as suggested.

.. jupyter-execute::

    # Experiment
    ns = 1e-9
    mu = 1e-6

    # qubit properties
    t1 = [45 * mu, 45 * mu]
    t2 = [value/2 for value in t1]

    # we will guess that our guess is 10% off the exact value of t1 for qubit 0.
    t1_estimated_shift = t1[0]/10

    # We use log space for the delays because of the noise properties
    delays = np.logspace(1, 11, num=20, base=np.exp(1))

>     delays = [delay * ns for delay in delays]

Reviewer: See comment in the test

Author: Changed as suggested.

>     # adding to the circuit 0 delay and long delays to help the curve fitting

Reviewer: 1. You're not adding these delays to "the circuit"; rather you're adding circuits with these delays.
2. You're not doing it in order to help the curve fitting (the curve fitting is doing fine without it), but because otherwise the fitting will provide wrong results (because the centers will be misplaced).

Author: Does this capture the essence of it?

```
# Adding circuits with delay=0 and long delays so the centers in the IQ plane won't be misplaced.  
# Without this, the fitting can provide wrong results.
```

    delays = np.insert(delays, 0, 0)

>     delays = np.append(delays, [t1[0]*2, t1[0]*3, t1[0]*4, t1[0]*5])

Reviewer: 1. Here and in the tests: Why do you add four delays, instead of only one?
2. Do you know why some of the letters are displayed in italics?

Author: 1.  I thought it will need extreme delays so I will get to the `0` center in the IQ plane and the curve fit need a number of extreme delays so the "direction" (slope) of the data will be easily deduced by the fitter. I checked it and only one is needed (I got carried away :sweat_smile:)
2. The italic letters are due to the asterisks. The string between the asterisks are in italics. The issue was resolved as I edited so we will add only one extreme delay.

changed to:
```
delays = np.insert(delays, 0, 0)
delays = np.append(delays, [t1[0]*2])
```
